app/src/commander.h

```diff
@@ -25,8 +26,10 @@ class CmdDeviceSetNameMsg;
 class CmdHandler
 {
 public:
+    std::unique_ptr<std::vector<PowerRelay>> m_pwrRelays;

```

Comment:
not needed in the public scope, change it back to private

Reply:
undoned